newm-server/src/main/kotlin/io/projectnewm/server/pugins/Monitoring.kt

```diff
@@ -0,0 +1,14 @@
+package io.projectnewm.server.pugins
+
+import io.ktor.server.application.*
+import io.ktor.server.plugins.*
+import io.ktor.server.request.*
+import org.slf4j.event.Level
+
+
+fun Application.configureMonitoring() {
```

Comment:
Do you know of any good monitoring solutions for ktor apps?

I can look into sentry.io and some others if we want to be able to track prod performance metrics

Reply:
Don't know. Let me know what you find out.